browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -48,6 +48,11 @@ int OnBeforeURLRequest_StaticRedirectWorkForGURL(
       URLPattern::SCHEME_HTTP | URLPattern::SCHEME_HTTPS, kCRXDownloadPrefix);
   static URLPattern autofill_pattern(
       URLPattern::SCHEME_HTTPS, kAutofillPrefix);
+  static URLPattern gvt1_pattern(
```

Comment:
maybe we should add comments and issue IDs to these patterns, for better understanding